force-app/main/default/classes/SOQL.cls

```diff
@@ -314,18 +353,28 @@ public with sharing class SOQL {
      */
     public static void cloneLineItems(Id origOppId, List<Id> cloneOppIds) {
         // Query the original Opportunity Line Items
-
+        List<OpportunityLineItem> originalOppLineItems =[SELECT Id, OpportunityId, PricebookEntryId, Product2Id, Quantity, TotalPrice FROM OpportunityLineItem WHERE OpportunityId = :origOppId];
         // Create a list to hold the new line items
-
+        List<OpportunityLineItem> newOppLineItems = new List<OpportunityLineItem>();
         // Generate a random number between 1 and 3
-
+        Integer randomQuantantiy = (Integer)Math.floor(Math.random() * 10) + 3;
```

Comment:
Two things here with this random number: (1) you don't use it below in your code (but should), (2) it doesn't generate a random number between 1 and 3. Instead it generates a random number between 3 and 12. 

Try this template in the future:
```
Integer randomNum = (Integer) (Math.random() * (max - min + 1)) + min;
Integer randomNum = (Integer) (Math.random() * (3 - 1 + 1)) + 1;
Integer randomNum = (Integer) (Math.random() * 3) + 1;
```